ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_activation.py

```diff
@@ -390,6 +390,17 @@ def test_paddle_celu(
     ),
 )
 def test_paddle_rrelu(*, dtype_and_x, on_device, fn_tree, frontend, test_flags):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
```

Comment:
Hi, Thanks for the quick PR! :)

Minor NIT: Please write the function args similar to other functions (in multiple lines instead of a single one).

Reply:
Done